### DISCUSSION TITLE
Get_Host and Get Application fails to pull Information for ids

### DISCUSSION THREAD

**Discussion by Odie71** [CrowdStrike/falconpy#920] [Q&A] [answered]
**Describe the bug**
After successful authentication, the following command is ran using the Uber Class
host_info = falcon.command("get_hosts", ids='REDACTED')

Status code is Successful
{'status_code': 200, 'headers': {'Server': 'nginx', 'Date': 'Fri, 17 Feb 2023 19:32:25 GMT', 'Content-Type': 'application/json', 'Content-Length': '219', 'Connection': 'keep-alive', 'Content-Encoding': 'gzip', 'Strict-Transport-Security': 'max-age=15724800; includeSubDomains, max-age=31536000; includeSubDomains', 'X-Cs-Region': 'us-2', 'X-Cs-Traceid': '5ae0d064-342f-47e3-8ea2-9e25478e98bd', ...}, 'body': {'meta': {...}, 'resources': [...], 'errors': [...]}}

It response a header and body.  The body shows
{'meta': {'query_time': 0.000410979, 'powered_by': 'discover-api', 'trace_id': '5ae0d064-342f-47e3-8ea2-9e25478e98bd'}, 'resources': [], 'errors': [{...}]}

The error return:
[{'code': 500, 'message': 'failed to get id=REDACTED'}]

**To Reproduce**

Authenticate to CrowdStrike Falcon
Use Uber Class to execute the get_host or get_applications function and provide valid ids

**Expected behavior**
Return the applications that are associated with the provided ids.

**Environment (please complete the following information):**
 - OS: Windows
 - Visual Studio Code
 - Python: Python 3.10.9
 - FalconPy: crowdstrike-falconpy 1.2.10

**Comment by jshcodes** [comment, marked as answer]
Hi @Odie71 -

A couple of follow up questions:

1. Host IDs for hosts returned from Discover are in the following format: `{CID}_{DEVICE_ID}`
    - If you update the ID in your search to match this format, are you returned a result?
2. Do you see results when you make a call to the `query_hosts` operation? (Example: `falcon.command("query_hosts")`)
